cloud-formation/membership-app.cf.yaml

```diff
@@ -240,6 +256,17 @@ Resources:
         FromPort: 443
         ToPort: 443
         CidrIp: 0.0.0.0/0
+  WazuhSecurityGroup:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: Allow outbound traffic from wazuh agent to manager
+      VpcId:
+        Ref: VpcId
+      SecurityGroupEgress:
+      - IpProtocol: tcp
+        FromPort: 1514
+        ToPort: 1515
+        CidrIp: 0.0.0.0/0
```

Comment:
presumably there's no sensible limit on the IP addresses?

Reply:
Hm that's a good question. We have dns set up for the service so I'm not 100% sure, but even if it is static it's considered private information so infosec advised us against publishing it in public repos. In the amigo recipe for example, we fetch it from SSM.